docker/base/setup_19.x

```diff
@@ -0,0 +1,368 @@
+#!/bin/bash
```

Comment:
Included this file here to avoid the antipattern of piping curl output to bash.

Reply:
Does this need to be in a third_party directory?